exampleAuthentication/1.authentication.js

```diff
@@ -64,16 +64,17 @@ console.log("entity1Puk pubk:", '04'+entity1Puk.getPublicKeyString().substr(2))
   let verifyAT = tokensFactory.tokens.verifyJWT(signedAT, "04"+entity1Puk.getPublicKeyString().substr(2))
   console.log('\tIs the signedJWT verified?', verifyAT)
 
+  const subject1Puk = Wallet.fromPrivateKey(subject1PrivateKey)
+  // '04' means uncompressed key (more info at https://github.com/indutny/elliptic/issues/138)
+  console.log("entity1Puk pubk:", '04'+subject1Puk.getPublicKeyString().substr(2))
 
-const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, "04"+subject1Pubk.getPublicKeyString().substr(2), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
+const alastriaSession = tokensFactory.tokens.createAlastriaSession(context, configData.didSubject1, subject1Puk.getPublicKeyString(), signedAT, tokenExpTime, tokenActivationDate, jsonTokenId)
```

Comment:
Just take the public key from the config file 
![image](https://user-images.githubusercontent.com/10101065/81929906-14ac2a00-95e8-11ea-86b4-8cde59b9248a.png)